docs/guides/insar_product_guide.md

```diff
@@ -37,15 +37,15 @@ For InSAR applications, processing is generally performed on the co-pol (VV or H
 
 ### Baselines
 #### Perpendicular Baseline
-The term *baseline* refers to the physical distance between the two vantage points from which images used as an InSAR pair are acquired. The baseline is decomposed into a normal and a perpendicular component as shown in Figure 2. 
+The term *baseline* refers to the physical distance between the two vantage points from which images used as an InSAR pair are acquired. The baseline is decomposed into perpendicular (also called normal) and parallel components, as shown in Figure 2. 
 
 To monitor surface deformation, the perpendicular baseline for the two acquisitions should be very small in order to maximize the coherence of the phase measurements. 
 
 In order to determine topography, two slightly different vantage points are required. Sensitivity to topography depends on the perpendicular baseline, the sensor wavelength, the distance between the satellite and the ground, and the sensor look angle. 
 
-![Figure 2](../images/baseline.png "Geometry of InSAR Baselines.")
+![Figure 2](../images/baseline_asf.png "Geometry of InSAR Baselines.")
 
-*Figure 2: Geometry of InSAR baselines. Two satellite passes image the same area on the ground from positions S1 and S2, resulting a baseline of B which can be decomposed into normal (B<sub>n</sub>) and perpendicular (B<sub>p</sub>) components. Here Y is the direction of travel, referred to as the ****along-track**** or ****azimuth**** direction, and X is the direction perpendicular to motion, referred to as the ****cross-track**** or ****range**** direction.*
+*Figure 2: Geometry of InSAR baselines. Two satellite passes image the same area on the ground from positions S<sub>1</sub> and S<sub>2 </sub>, resulting in a baseline of B, which can be decomposed into ****perpendicular**** (B<sub>⟂ </sub>) and ****parallel**** (B<sub>∥ </sub>) components. Here Y is the direction of travel, referred to as the ****along-track**** or ****azimuth**** direction, and X is the direction perpendicular to motion, referred to as the ****cross-track**** or ****range**** direction. Credit: ASF*
```

Comment:
are the trailing spaces in 3/4 of the <sub> tags intentional?

Reply:
@asjohnston-asf Yes it was intentional. The way the italicized unicode interacts with the closing bracket makes it very hard to read when rendered on the website. Same for the comma after the 2. I found it much more visually pleasing with the small spaces, but let me know if there's a better approach or if that could introduce problems I'm not thinking of.